docs/guide/multisite.md

```diff
@@ -2,109 +2,202 @@
                     # being outside of the subtree starting at root_location.
                     # Default value is an empty array.
                     # Prefixes are not case sensitive.
                     excluded_uri_prefixes: [ /media, /images ]
 ```
 
-##### `content.tree_root.location_id`
+### Configuration parameters
 
-This parameter sets the Location ID of the content root. The API is jailed within this root, and nothing above this level will be accessible by default. This parameter can be filtered using the `excluded_uri_prefixes` parameter described below.
+The two configuration parameters, `location_id` and `excluded_uri_prefixes` are taken into account in several places,
+for example in URL alias lookups for incoming web requests, and [URL alias path generation in Twig templates](content_rendering.md#adding-links).
+In addition you need to consider them when generating site paths in your layout, or other places rendering site/tree structure.
 
-##### `content.tree_root.excluded_uri_prefixes`
+#### `location_id`
 
-Use this parameter to exclude a list of URIs from the root jail defined using `location_id` described above. Add as many locations pathString as you like. In the example above, the Media and Images folders will be accessible using their own URI even though they're not below `tree_root.location_id`.
+`content.tree_root.location_id` sets the Location ID of the content root of your site
+and nothing above this level will be accessible by default.
+This parameter can be filtered using the `excluded_uri_prefixes` parameter.
 
-!!! note
+#### `excluded_uri_prefixes`
 
-    Leading slashes (`/`) are automatically trimmed internally by eZ Platform, so they can be ignored.
+`content.tree_root.excluded_uri_prefixes` defines which URIs ignore the root limit set using `location_id`.
+Add as many Location pathStrings as you like.
+In the example above, the Media and Images folders will be accessible using their own URI even though they are not below `content.tree_root.location_id`.
 
-## Usage
+!!! note
 
-### Multisite Design
+    Leading slashes (`/`) are automatically trimmed internally, so they can be ignored.
 
-eZ Platform does not apply a [Legacy template fallback](https://doc.ez.no/display/EZP/Legacy+template+fallback) like eZ Publish did. You can, however, have different designs in your multisite installation if you organize the view configuration with the use of siteaccesses.
+## Configuration example
 
-You can see how this is done on the example of a multisite that contains two blogs with different designs, but sharing some templates. Let's assume you have a siteaccess called `second_site`. You can organize your templates in the following folder structure:
+To see how multisite can be used, let's look at an example of two sites using the same eZ Platform instance: a general company site and a site for a specific event.
+Separate SiteAccess groups are set up for the two sites:
```

Comment:
Any reason why we opted for separate groups here? It1s valid, but if the two sites share config it's more common if they share group as well. On the other side if you plan to translate the times then groups are indeed good to represent the sites, while access will represent language.

So in sum: Maybe somehow explain why we opt for using separate group here for those that might wonder.

Reply:
No particular reason for using groups except me being too copy-paste-happy.